src/ToDoList.js

```diff
@@ -18,107 +16,97 @@ import {
   IconButton,
 } from "@mui/material";
 
-const containerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  width: "500px",
-  height: "600px",
-  backgroundColor: "lightGrey",
-  borderRadius: "30px",
-  margin: "0 auto",
-  marginTop: "30px",
-};
-
-const titleStyle = {
-  color: "black",
-  variant: "h1",
-  width: "300px",
-  height: "50px",
-  //borderRadius: "20px",
-  //backgroundColor: "lightBlue",
-  margin: "0 auto",
-  fontWeight: "bold",
-  //justifyContent: "center",
-  //alignItems: "center",
-};
-
-const inputContainerStyle = {
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-  //height: "40px",
-};
-
-/*const taskItemStyle = {
-  width: "200px",
-  display: "flex",
-  display: "flex",
-  justifyContent: "center",
-  alignItems: "center",
-};*/
-
-const completedTaskStyle = {
-  textDecoration: "line-through",
-  textDecorationStyle: "solid",
-  textDecorationThickness: "2px",
-  color: "black",
-};
-
-/*const deleteButtonStyle = {
-  fontSize: "10px",
-  padding: "3px 3px",
-};*/
+const containerStyle = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  flex-direction: column;
+`;
+
+const titleStyle = css`
+  color: black;
+  font-size: 24px;
+  width: 300px;
+  height: 50px;
+  border-radius: 20px;
+  background-color: lightBlue;
+  margin: 0 auto;
+  font-weight: bold;
+  margin-top: 30px;
+  display: flex;
+  flex-direction: column;
+`;
+
+const inputContainerStyle = css`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-top: 30px;
+`;
+
+const completedTaskStyle = css`
+  text-decoration: line-through;
+  text-decoration-style: solid;
+  text-decoration-thickness: 2px;
+  color: black;
+`;
 
 const ToDoList = () => {
+  const [tasks, setTasks] = useState([]);
+  const [taskInfo, setTaskInfo] = useState("");
+  const [filter, setFilter] = useState("all");
+
   useEffect(() => {
     const storedTasks = localStorage.getItem("tasks");
```

Comment:
Not sure if this `useEffect` is needed, as you maybe can set the tasks from the local storage as the initial value for the `tasks` variable when you use `useState`

Reply:
I checked this and see that this indeed works as I implemented the local storage inside the additional functions that change the state of the task. Now the tasks are set from the local storage as the initial value.

Reply:
Ok, but I don't see that you removed the `useEffect` then.